packages/utilities/psammead-test-helpers/src/index.js

```diff
@@ -1,11 +1,12 @@
 import renderer from 'react-test-renderer';
```

Comment:
is it possible to use `render` instead of `renderer` here as well (the same as you done below), then we don't have to import `react-test-renderer`: https://github.com/bbc/psammead/blob/13307abf6cda16bf55203aab604e04c731ba5853/packages/utilities/psammead-test-helpers/src/index.js#L30

Reply:
also would it be good to add `@testing-library/react` as a dependency of this package ?

https://github.com/bbc/psammead/blob/13307abf6cda16bf55203aab604e04c731ba5853/packages/utilities/psammead-test-helpers/package.json#L27-L29

Reply:
@thekp `react-test-renderer` is used in a shallow render function and `@testing-library/react` doesn't do shallow rendering for [these reasons](https://kentcdodds.com/blog/why-i-never-use-shallow-rendering). The best thing to do would be to refactor tests using shallow rendering to remove the use of shallow rendering.

Oh I didn't realise it had to be a dependency of this package too. Should it not be a dev dependency though?

Reply:
I just had a look to see which tests are using shallow rendering. There appears to be no tests using shallow rendering. It's worth considering removing the shallow render test helpers.

Reply:
oh yeah, making it a dev-dependency makes more sense 👍